python/cuml/test/test_fil.py

```diff
@@ -194,7 +194,7 @@ def test_fil_regression(n_rows, n_columns, num_rounds, tmp_path, max_depth):
 @pytest.mark.parametrize('n_columns', [30])
 @pytest.mark.parametrize('n_estimators', [1, 10])
 @pytest.mark.parametrize('max_depth', [2, 10, 20])
-@pytest.mark.parametrize('n_classes', [2, 5, 25])
+@pytest.mark.parametrize('n_classes', [2, 5, 10])
```

Comment:
@canonizer didn't we want at least one test with >>5 classes? Wondering if it can be run only on e.g. max_depth = 2 && n_estimators == 1 && storage_type = False ?

Reply:
Since LightGBM uses grove-per-class method, `n_classes=25` and `n_estimators=10` will produce 250 trees, leading to long training time. As a compromise, can we settle for `n_classes=15`?

Reply:
What is the performance difference between 10, 15 and 25 classes?

Reply:
LightGBM fits `[num_class] * [num_round]` trees, so higher number of class will result into longer training time.

Reply:
Per @levsnv's request, we will keep testing the `n_classes=25` case. To reduce test time, we will only test `n_classes=25` with `n_estimators=1`.

Reply:
Edited comments above, since `test_fil_skl_classification` test is using `GradientBoostingClassifier`, not LightGBM.

Reply:
See my latest commit.